### PR TITLE
Fixes #31682 - Remove puppet and ostree repos

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
TODO: 
* [ ] properly cleanup candlepin ostree content

File: lib/katello/tasks/upgrades/4.0/remove_ostree_content.rake [hunk 1]
```diff
@@ -0,0 +1,16 @@
+namespace :katello do
+  namespace :upgrades do
+    namespace '4.0' do
+      desc "Removes ostree content from ."
```
Comment (jjeffers) [inline lib/katello/tasks/upgrades/4.0/remove_ostree_content.rake:4]: I was confused by the decription "Removes ostree content" but it doesn't mention puppet content. Is that intentional?

Comment (jlsherrill) [reply lib/katello/tasks/upgrades/4.0/remove_ostree_content.rake:4]: yes, i will fix that

Comment (jlsherrill) [reply lib/katello/tasks/upgrades/4.0/remove_ostree_content.rake]: updated!

**Review by jjeffers — APPROVED** [review]
Tested this by creating products, repositorie, and CVVs with combinations of yum, puppet, and ostree content. After runing the rake task and migration, the ostree and puppet repositiories were removed.

LGTM

**Comment by chris1984** [comment]
[test katello]

**Review by chris1984 — APPROVED** [review]

**Pull request merged**